apps/src/gamelab/GameLab.js

```diff
@@ -174,7 +256,7 @@ GameLab.prototype.callHandler = function (name, allowQueueExtension, extraArgs)
     } else {
       // TODO (cpirich): support events with parameters
       if (handler.name === name) {
-        this.JSInterpreter.queueEvent(handler.func, extraArgs);
+        handler.func.apply(null, extraArgs);
```

Comment:
This line is failing in https://staging-studio.code.org/s/starwars/stage/1/puzzle/15 whenever I try to move r2-d2 in any direction (via buttons or keys):

```
Uncaught TypeError: handler.func.apply is not a function(anonymous function) @ studio.js:790callHandler @ studio.js:773Studio.onTick @ studio.js:979
11studio.js:790 Uncaught TypeError: handler.func.apply is not a function(anonymous function) @ studio.js:790callHandler @ studio.js:773Studio.onTick @ studio.js:1002
```

I'm going to revert this PR so we don't deploy to prod and to give @cpirich time to investigate.